src/main/java/org/apache/maven/plugins/clean/CleanMojo.java

```diff
@@ -193,6 +203,8 @@ public void execute()
             return;
         }
 
+        String multiModuleProjectDirectory = session != null
```

Comment:
Can the session be now truly `null`?

Reply:
It will never be null in practice, but in theory it can, and it is actually null during tests.

Reply:
So the `@Requirement` will inject `null` during testss?

Reply:
During unit tests, yes, `session` is `null`.